salt/metalk8s/kubernetes/etcd/installed.sls

```diff
@@ -80,3 +80,24 @@ Create local etcd Pod manifest:
     - require:
       - file: Create etcd database directory
       - file: Ensure etcd CA cert is present
+
+# wait until etcd is actually up and running
+Delay after etcd pod is deployed:
+  module.wait:
+    - test.sleep:
+      - length: 10
+    - watch:
+      - metalk8s: Create local etcd Pod manifest
+
+Wait for etcd:
+  http.wait_for_successful_query:
+    - name: https://127.0.0.1:2379/health
+    - verify_ssl: True
+    - ca_bundle: /etc/kubernetes/pki/etcd/ca.crt
+    - cert:
+      - /etc/kubernetes/pki/etcd/server.crt
+      - /etc/kubernetes/pki/etcd/server.key
+    - status: 200
+    - match: '{"health":"true"}'
+    - require:
+      - module: Delay after etcd pod is deployed
```

Comment:
If you do this here you can basically remove the `metalk8s.kubernetes.etcd.healthy` sls (and change it in `metalk8s.orchestrate.etcd`)